packages/combobox/README.md

```diff
@@ -21,8 +23,136 @@ When looking to leverage the `Combobox` base class as a type and/or for extensio
+    <sp-menu-item value="blue">Blue</sp-menu-item>
+</sp-combobox>
+```
+
+</sp-tab-panel>
+<sp-tab value="xl">Extra Large</sp-tab>
+<sp-tab-panel value="xl">
+
+```html
+<sp-combobox size="xl" label="Color">
+    <sp-menu-item value="red">Red</sp-menu-item>
+    <sp-menu-item value="green">Green</sp-menu-item>
+    <sp-menu-item value="blue">Blue</sp-menu-item>
+</sp-combobox>
+```
+
+</sp-tab-panel>
+</sp-tabs>
+
+## Labeling
+
+A combobox must be labeled.
+Typically, you should render a visible label via `<sp-field-label>`.
+For exceptional cases, provide an accessible label via the `label` attribute.
+
+```html
+<sp-field-label for="color">Color</sp-field-label>
+<sp-combobox id="color">
+    <sp-menu-item value="red">Red</sp-menu-item>
+    <sp-menu-item value="green">Green</sp-menu-item>
+    <sp-menu-item value="blue">Blue</sp-menu-item>
+</sp-combobox>
+```
+
+## Providing options
+
+Combobox options are presented as a popup menu. Menu items can be provided via markup as `<sp-menu-item>` children, or by assigning an array to the `options` property of an `<sp-combobox>`.
+
+### Options property
+
+Instead of providing `<sp-menu-item>` children, you can assign an array of `ComboboxOptions` to the `options` property, and `<sp-combobox>` will create matching menu items:
+
+```html-no-demo
+<sp-combobox id="color" label="Color"></sp-combobox>
+
+<script>
+    document.getElementById('color').options = [
+        { value: "red", itemText: "Red" },
+        { value: "green", itemText: "Green" },
+        { value: "blue", itemText: "Blue" }
+    ];
+</script>
+```
+
+### Dynamic options
+
+When you update the `options` property or add/remove `<sp-menu-item>` children, the `<sp-combobox>` will detect that change and update its popup menu contents. For example, using [Lit](https://lit.dev/):
+
+```ts
+render() {
+    return html`<sp-combobox label="Color" .options=${this.colorOptions}></sp-combobox>`;
+}
+
+mutate() {
+    this.colorOptions = [
+        ...this.colorOptions,
+        { value: 'purple', itemText: 'Purple' }
+    ];
+}
+```
+
+## Autocomplete
+
+The text in an `<sp-combobox>` is editable, and the string the user has typed in will become the `value` of the combobox unless the user selects a different value in the popup menu.
+
+### None
+
+`autocomplete="none"`
+
+The suggested popup menu items will remain the same regardless of the currently-input value. Whenever the currently-typed input exactly matches the `value` of a popup menu item, that item is automatically selected.
+
+### List
+
+`autocomplete="list"`
+
+The popup menu items are filtered to only those completing the currently-input value.
+
+## Focus and Accessibility
+
+The combobox supports both mouse and keyboard navigation. Mobile behavior is currently unspecified.
+
+When an `<sp-combobox>` is focused, pressing the down arrow moves focus to the first menu item in the popup menu.
+The up and down arrows then move between available menu items.
```

Comment:
There's a couple of new lines in this section that will be merged into single `<p>` tags here, are we OK with that?

Reply:
Yeah I'm just putting newlines for easier reading of the source MD. Happy to merge onto a single line if you prefer.